docs/contributing/README.md

```diff
@@ -118,6 +120,41 @@ Then, run the continuous batching tests:
 python -m pytest -v -x tests/e2e -m cb
 ```
 
+## Debugging
```

Comment:
should debugging be in its own file so there's a nav item called debugging that readers can easily find?

Reply:
Yeah, I thought about it, technically you are debugging only when you plan on contributing 😄 I'm fine either way.

Reply:
> technically you are debugging only when you plan on contributing 😄

True true, there is a fine line between:
- debugging -> Something isn't working and I'm being paid to find out why and contribute a fix
- troubleshooting -> Something isn't working and I'm being paid to find a workaround

(I'm fine leaving this as is and we can evolve as needed as we add more to the docs)